Remora.Commands/Trees/CommandTreeBuilder.cs

```diff
@@ -54,6 +54,18 @@ public class CommandTreeBuilder
             }
         }
 
+        /// <summary>
+        /// Registers a module type with the builder.
+        /// </summary>
+        /// <param name="commandModule">The module type.</param>
+        public void RegisterModule(Type commandModule)
```

Comment:
It may be worth just calling this method from `RegisterModule<TModule>` to simplify the code :)